src/main.ts

```diff
@@ -52,7 +45,9 @@ export async function run(): Promise<void> {
 
     try {
       const runResult = await runResultPromise;
-      const success = runResult.results.every(x => x.success);
+      const success = runResult.results.every(
+        x => x.status === TestStatus.PASSED
```

Comment:
```suggestion
        x => x.status === 'PASSED' || x.status === 'SKIPPED'
```
Is this what we want instead?

Reply:
I'm including `FLAKY` there too